tests/BaGet.Core.Tests/BaGet.Core.Tests.csproj

```diff
@@ -1,11 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\BaGet.Core\BaGet.Core.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="coverlet.collector" Version="3.1.2">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
Why did you add an explicit dependency to coverlet in the test projects? Isn't this already added by the `Direction.Build.props` file?

Reply:
I missed it while doing autoupdate with `dotnet outdated` tool, seems like this is caused by it's behavior.

Reverted